internal/kafka/internal/services/clusters.go

```diff
@@ -47,6 +47,7 @@ type ClusterService interface {
 	GetClientID(clusterID string) (string, error)
 	ListGroupByProviderAndRegion(providers []string, regions []string, status []string) ([]*ResGroupCPRegion, *apiErrors.ServiceError)
 	RegisterClusterJob(clusterRequest *api.Cluster) *apiErrors.ServiceError
+	DeregisterClusterJob(clusterId string) *apiErrors.ServiceError
```

Comment:
Accronyms like 'id` should be written in uppercase. Therefore in this case it should be `clusterID string`